crypto/src/main/kotlin/web5/sdk/crypto/Algorithm.kt

```diff
@@ -0,0 +1,26 @@
+package web5.sdk.crypto
+
+/**
+ * The supported JSON Web Signature algorithms.
+ */
+public enum class Algorithm {
+  /** ECDSA using secp256k1 curve and SHA-256. */
+  ES256K,
+
+  /** Edwards-curve Digital Signature Algorithm. */
+  EdDSA;
+
+  internal fun toNimbusdsJWSAlgorithm(): com.nimbusds.jose.JWSAlgorithm {
+    return when (this) {
+      ES256K -> com.nimbusds.jose.JWSAlgorithm.ES256K
+      EdDSA -> com.nimbusds.jose.JWSAlgorithm.EdDSA
+    }
+  }
+}
+
+/**
+ * Converts an [Algorithm] from the nimbusds library to a [Algorithm].
+ */
+public fun com.nimbusds.jose.Algorithm?.toWeb5Algorithm(): Algorithm? {
```

Comment:
Well the naming is confusing enough but maybe this should just be called 

toAlgorithm(). Kind of strange to have web5 in the name

Reply:
jwt.header.algorithm.toAlgorithm() would be more ugly probably haha

Reply:
Yeah I'm not thrilled about the name either :(

I think `toWeb5Algorithm` is the "less bad". Also, I think we're considering a different interface from the conversation in https://github.com/TBD54566975/web5-rs/issues/38